modules/iam-assumable-role-with-oidc/main.tf

```diff
@@ -43,7 +45,7 @@ data "aws_iam_policy_document" "assume_role_with_oidc" {
 
     content {
       effect  = "Allow"
-      actions = ["sts:AssumeRoleWithWebIdentity"]
```

Comment:
leave this and just add `"sts:TagSessions"`

Reply:
So _always_ have the `sts:TagSessions` permission aswell?

Reply:
yes